### DISCUSSION TITLE
6주차 과제 제출

### DISCUSSION THREAD

**Discussion by youngsangcho** [youngsangcho/generativedesign2_2021#24] [Announcements]
6주차 과제 제출하지 않으신 분들이 많네요.
늦어도 일요일까지는 제출하시기 바랍니다-

**Comment by SeyounJung01** [comment]
정세윤
https://blog.naver.com/seyounjj2001/222556051973

**Comment by imyeeunok** [comment]
rㅣㅁ예은

https://blog.naver.com/010312kye/222556070898

**Comment by Joeun410** [comment]
김조은 
https://blog.naver.com/PostView.naver?blogId=joanne3456&Redirect=View&logNo=222556078519&categoryNo=13&isAfterWrite=true&isMrblogPost=false&isHappyBeanLeverage=true&contentLength=34460

**Comment by jwejwepark** [comment]
https://blog.naver.com/glqkfl0726/222556144398
